gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -24,7 +24,20 @@
 #ifndef __NNS_PLUGIN_API_FILTER_H__
 #define __NNS_PLUGIN_API_FILTER_H__
 
-#include "tensor_typedef.h"
+#include "nnstreamer_plugin_api.h"
```

Comment:
nnstreamer_plugin_api is for nnstreamer plugin (gst element) writers.
nnstreamer_plugin_api_filter is for nnstreamer subplugin writers.

Thus, please do NOT include ```nnstreamer_plugin_api.h```.


We reallly need to clean up devel packages. (separate subplugins-devel from gst-devel)

Reply:
Then, I will remove this change.